api.go

```diff
@@ -423,7 +424,8 @@ func DeleteContainer(podID, containerID string) (*Container, error) {
 	}
 
 	// Delete it.
-	err = c.delete()
+	cImpl := c.(*Container)
```

Comment:
Not needed if we agree that `fetchContainer()` should not return `VCContainer` type.